.dependabot/config.yml

```diff
@@ -4,6 +4,31 @@ update_configs:
   - package_manager: 'javascript'
     directory: '/'
     update_schedule: 'weekly'
+    automerged_updates:
+      - match:
+          dependency_name: 'chai*'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'cypress'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'eslint*'
```

Comment:
Note the wildcards on some of these. For example, this will match `eslint`, but also `eslint-config-prettier`, `eslint-plugin-standard` etc